website/content/docs/commands/ui.mdx

```diff
@@ -22,10 +22,22 @@ will be opened in the default browser.
 An identifier may be provided, in which case the UI will be opened to view the
 details for that object. Supported identifiers are jobs, allocations and nodes.
 
+If ACLs are enabled, the web UI will start in an unauthenticated state and you
+may see a 403 Unauthorized page if anonymous read access is denied. The `nomad
+ui -authenticate` option will exchange your command line client's Nomad ACL
+token for a one-time token to the web UI. That one-time token will be
```

Comment:
I see that this was already reviewed in #10097 but is there a word missing here before `to`? Or maybe it should be `for the web UI`?

Reply:
Yeah, that is awkward. What about making it even more explicit:

> The `nomad ui -authenticate` option will exchange your command line client's Nomad ACL token for a one-time token, which is passed to the web UI.

Reply:
excellent, thank you!

Reply:
👍  Done in f4621f2